scripts/partest-check-files.sc

```diff
@@ -0,0 +1,77 @@
+import $ivy.`com.lihaoyi::ammonite-ops:2.3.8`, ammonite.ops._, mainargs._
+import java.io.File
+
+@main(
+  doc = """" +
+  "Tool used to check integrity of files defined in partest tests and 
+  actually defined in partest repo. Allow to check which blacklisted files
+  are not existing and can suggest correct blacklisted item name.
+  Also checks for duplicates in blacklisted items.""")
```

Comment:
This script should be executed in the CI, to make sure we don't commit corrupted lists.